cypress/e2e/externalLinks/english-external-links-validator.cy.js

```diff
@@ -19,6 +19,7 @@ const excludedlinks = [
   'https://sosmt.gov/elections/vote/?ref=voteusa_en',
   'https://www.vivote.gov/voters/register-to-vote/?ref=voteusa_en',
   'https://www.vivote.gov/voters/voter-lookup/?ref=voteusa_en',
+  'https://www.elections.ny.gov/votingregister.html?ref=voteusa_en',
```

Comment:
Add `https://voterlookup.elections.ny.gov/?ref=voteusa_en`

Reply:
This has been resolved